<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>266</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="qStackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tauler">
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>353</width>
          <height>193</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="lineEdit_13"/>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="lineEdit_10"/>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEdit_9"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_11"/>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="lineEdit_15"/>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="lineEdit_14"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_8"/>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="lineEdit_12"/>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="lineEdit_7"/>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>10</y>
          <width>82</width>
          <height>164</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>77</width>
             <height>49</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_validar">
           <property name="text">
            <string>Validar</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>77</width>
             <height>49</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_reiniciar">
           <property name="text">
            <string>Reiniciar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="configuracio">
       <widget class="QLabel" name="lbl_imatge">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>381</width>
          <height>431</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../../../../.designer/backup/haunting.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>40</y>
          <width>138</width>
          <height>60</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Valor mínim:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_min"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Valor màxim:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_max"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="quanta">
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>0</y>
          <width>461</width>
          <height>211</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="lbl_quanta_titol">
           <property name="text">
            <string>Quadrat màgic</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lbl_quanta_desc">
           <property name="text">
            <string>Totes les files i columnes han de sumar el mateix</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lbl_quanta_credits">
           <property name="text">
            <string>2024. Institut Marianao</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>477</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFitxer">
    <property name="title">
     <string>Fitxer</string>
    </property>
    <addaction name="actionTauler"/>
    <addaction name="actionConfiguracio"/>
    <addaction name="actionSortir"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionQuant_a"/>
   </widget>
   <addaction name="menuFitxer"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionQuant_a">
   <property name="text">
    <string>Quant a</string>
   </property>
  </action>
  <action name="actionTauler">
   <property name="text">
    <string>Tauler</string>
   </property>
  </action>
  <action name="actionConfiguracio">
   <property name="text">
    <string>Configuració</string>
   </property>
  </action>
  <action name="actionSortir">
   <property name="text">
    <string>Sortir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
